db/mongo_test.go

```diff
@@ -0,0 +1,302 @@
+package db
+
+import (
+	"log"
+	"net/url"
+	"os"
+	"sort"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"gopkg.in/mgo.v2"
+	"gopkg.in/mgo.v2/bson"
+)
+
+var (
+	trialCreateTime = time.Date(2014, time.November, 12, 11, 45, 26, 371000000, time.UTC)
+	trialTimeZone   = "America/Los_Angeles"
+)
+
+var cron1 = CronJob{
+	ID:       "",
+	IsActive: true,
+	Function: "test1",
+	Workload: "",
+	CronTime: "* * * * * *",
+	TimeZone: trialTimeZone,
+	Created:  trialCreateTime,
+}
+
+var cron2 = CronJob{
+	ID:       "",
+	IsActive: true,
+	Function: "test1",
+	Workload: "",
+	CronTime: "1 2 3 4 5 6",
+	TimeZone: trialTimeZone,
+	Created:  trialCreateTime,
+}
+
+var cron3 = CronJob{
+	ID:       "",
+	IsActive: false,
+	Function: "test2",
+	Workload: "",
+	CronTime: "6 5 4 3 2 1",
+	TimeZone: trialTimeZone,
+	Created:  trialCreateTime,
+}
+
+var cron4 = CronJob{
+	ID:       "",
+	IsActive: true,
+	Function: "test3",
+	Workload: "",
+	CronTime: "6 5 4 3 2 1",
+	TimeZone: trialTimeZone,
+	Created:  trialCreateTime,
+}
+
+var cron5 = CronJob{
+	ID:       bson.NewObjectId().Hex(),
+	IsActive: true,
+	Function: "test4",
+	Workload: "",
+	CronTime: "6 5 4 3 2 1",
+	TimeZone: trialTimeZone,
+	Created:  trialCreateTime,
+}
+
+var cron6 = CronJob{
+	ID:       bson.NewObjectId().Hex(),
+	IsActive: true,
+	Function: "test4",
+	Workload: "{\"foo\":\"bar\"}",
+	CronTime: "6 5 4 3 2 1",
+	TimeZone: trialTimeZone,
+	Created:  trialCreateTime,
+}
```

Comment:
in general data shared between tests is harder to read, and makes tests harder to modify

I think this file would be a lot more readable if each test declared its own data inside the scope of the test function.